daemon/snap.go

```diff
@@ -206,6 +228,34 @@ func clientHealthFromHealthstate(h *healthstate.HealthState) *client.SnapHealth
 	}
 }
 
+func isSnapMonitored(st *state.State, instanceName string) (bool, error) {
```

Comment:
this feels like it should be a helper in snapstate

Reply:
there is one, we just need to make it exported

Reply:
thank you, updated